index.html

```diff
@@ -5,24 +5,151 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Celine Vogel Personal Website</title>
+<link rel="stylesheet" href="styles.css">
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300&display=swap');
+  </style> 
 </head>
 <body>
 <header>
     <h1>Celine Vogel</h1>
-    <nav>
-        <a href="#contact">Contact</a>
-        <a href="#biography">Biography</a>
+
+    <br>
+
+    <nav class="main-nav">
+        <a href="#about me" class="button">About me</a>
+        <a href="#abilities" class="button">Abilities</a>
+        <a href="#interests" class="button">Interests</a>
+        <a href="#contact" class="button">Contact</a>
     </nav>
+
+    <br>
+
 </header>
+
 <figure>
-<img src="https://source.unsplash.com/random/600x300" alt="">
-<figcaption>Random image, that changes every time you reload.</figcaption>
+    <img src="https://photostockeditor.com/asset/1080/35/359a/PSE371.jpg" class="image" alt="Picture of a Laptop in the dark.">
 </figure>
-<section id="contact">
-  <h2>Contact</h2>
+
+<br>
```

Comment:
I think Jerry told us not to use this form for spaces and should solve this with CSS :)